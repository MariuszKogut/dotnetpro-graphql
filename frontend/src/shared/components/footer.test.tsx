import React from 'react'
import { render } from '@testing-library/react'
import Footer from './footer'

describe('Footer', () => {
  it('renders without errors', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
})
