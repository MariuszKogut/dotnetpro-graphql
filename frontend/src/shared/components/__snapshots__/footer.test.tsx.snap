// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without errors 1`] = `
<div>
  <footer
    class="p-3"
  >
    <nav
      class="navbar navbar-expand navbar-light bg-light fixed-bottom"
    >
      <div
        class="mr-auto navbar-nav"
      >
        Â© 
        2020
      </div>
    </nav>
  </footer>
</div>
`;
